<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waiting</class>
 <widget class="QWidget" name="Waiting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(17, 171, 120);
color: white;</string>
  </property>
  <widget class="QLabel" name="waitingGif">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="outNav">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
border: 2px solid white;
background-color: transparent;
color: white;
}


QPushButton:hover {
	background-color: white;
color:  rgb(42, 74, 74);
}

QPushButton:pressed {
	background-color: #f0f0f0;
}</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="roomId">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>RoomID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please wait until the exam start...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
